### DISCUSSION TITLE
How can I add an item directly from TMDB, IGDB, etc.

### DISCUSSION THREAD

**Discussion by TheWilley** [IgnisDa/ryot#367] [Q&A] [answered]
So far I've figured out how to:
* Add entries manually (trough the button on the homepage)
* Delete an entry (#321) 

Now, once again, I hope I have not missed something incredibly obvious here, but I cannot for the life of me figure out how items are added from online sources such as IGDB or TMDB like they are in the [demo website](https://ryot.fly.dev/). Are they imported via my accounts (trough [variables](https://ignisda.github.io/ryot/configuration.html) provided before building) or do I need to use [integrations](https://ignisda.github.io/ryot/integrations.html)? How does this work for video games and other media without integrations? Is data fetched automatically eventually after adding an item manually? 

To clarify, I mean items which have "TMBD:Show" or similar in the top left corner. How was that data fetched?

![bild](https://github.com/IgnisDa/ryot/assets/89783791/216780fc-ce98-4f7c-bc65-8ceb863c565e)

I want to apologise beforehand for flooding the github discussion page with many questions.

**Comment by IgnisDa** [comment, marked as answer]
You just need to search for the media (for example a show). Once you select what you want, click on it and the data will be downloaded to your database. LMK if you need more help.

**Reply by TheWilley** [reply]
I totally missed that there is a search function, that's my bad! Thanks for the quick response!
